src/incidents/view/ViewIncident.tsx

```diff
@@ -31,72 +31,74 @@ const ViewIncident = () => {
       dispatch(fetchIncident(id))
     }
   }, [dispatch, id])
-
-  return (
-    <>
-      <Row>
-        <Column>
-          <div className="form-group incident-date">
-            <h4>{t('incidents.reports.dateOfIncident')}</h4>
-            <h5>{format(new Date(incident?.date || ''), 'yyyy-MM-dd hh:mm a')}</h5>
-          </div>
-        </Column>
-        <Column>
-          <div className="form-group incident-status">
-            <h4>{t('incidents.reports.status')}</h4>
-            <h5>{incident?.status}</h5>
-          </div>
-        </Column>
-        <Column>
-          <div className="form-group incident-reported-by">
-            <h4>{t('incidents.reports.reportedBy')}</h4>
-            <h5>{incident?.reportedBy}</h5>
-          </div>
-        </Column>
-        <Column>
-          <div className="form-group incident-reported-on">
-            <h4>{t('incidents.reports.reportedOn')}</h4>
-            <h5>{format(new Date(incident?.reportedOn || ''), 'yyyy-MM-dd hh:mm a')}</h5>
-          </div>
-        </Column>
-      </Row>
-      <div className="border-bottom mb-2" />
-      <Row>
-        <Column md={12}>
-          <TextInputWithLabelFormGroup
-            label={t('incidents.reports.department')}
-            name="department"
-            value={incident?.department}
-          />
-        </Column>
-      </Row>
-      <Row>
-        <Column md={6}>
-          <TextInputWithLabelFormGroup
-            name="category"
-            label={t('incidents.reports.category')}
-            value={incident?.category}
-          />
-        </Column>
-        <Column md={6}>
-          <TextInputWithLabelFormGroup
-            label={t('incidents.reports.categoryItem')}
-            name="categoryItem"
-            value={incident?.categoryItem}
-          />
-        </Column>
-      </Row>
-      <Row>
-        <Column md={12}>
-          <TextFieldWithLabelFormGroup
-            label={t('incidents.reports.description')}
-            name="description"
-            value={incident?.description}
-          />
-        </Column>
-      </Row>
-    </>
-  )
+  if (incident) {
```

Comment:
We have a loading status as part of the incident slice that we should add into this check as well.